webpack/assets/javascripts/react_app/components/ToastsList/ToastList.scss

```diff
@@ -0,0 +1,9 @@
+/* Make space for <header> component */
+.foreman-toast {
+  top: 1em;
+}
+
+/* Make toasts visible over modal (z-index: 1050) */
+.pf-c-alert-group {
+  z-index: 1100 !important;
```

Comment:
@laviro could you change it in https://github.com/theforeman/foreman/pull/8619/files to a variable?

Reply:
as we discussed in https://github.com/theforeman/foreman/pull/8629#discussion_r665913992
there's no need to add it in the new confirm-modal